Add Query Insights excluded_indices section (#10011)
opensearch-project/documentation-website: _observing-your-data/query-insights/top-n-queries.md

}
```
{% include copy-curl.html %}

> ## Excluding indices from top N queries

Reviewer: ```suggestion
## Excluding indexes from top N queries
```

You can exclude search queries from the top N query list based on the indexes they target. This is useful when certain indexes are known to have long-running queries and don't need to be monitored.

> A query will be excluded if it searches any shard that belongs to an index listed in `excluded_indices`.

Reviewer: ```suggestion
A query is excluded if it searches any shard that belongs to an index listed in `excluded_indices`.
```
